helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -646,6 +645,7 @@ service:
         {{- if .Values.environment }}
         - resource/add_environment
         {{- end }}
+        - batch
```

Comment:
Please move it right after filter processor. There is no performance improvement in moving it further